authui/src/authflowv2/components/settings-title.css

```diff
@@ -3,12 +3,12 @@
     --settings-title__font-family: var(
       --typography-headline-small__font-family
     );
-    --settings-title__font-size: var(--typography-headline-small__font-size);
-    --screen-title__line-height: var(--typography-headline-small__line-height);
+    --settings-title__font-size: var(--typography-title-large__font-size);
+    --screen-title__line-height: var(--typography-title-large__line-height);
     --screen-title__letter-spacing: var(
-      --typography-headline-small__letter-spacing
+      --typography-title-large__letter-spacing
     );
-    --screen-title__font-weight: var(--typography-headline-small__font-weight);
+    --screen-title__font-weight: var(--typography-title-large__font-weight);
```

Comment:
This is too surprising. We do not expect a file named as `settings-title.css` to have declaration on `--screen-title`. What are you trying to achieve here?